doc/install_mne_c.rst

```diff
@@ -2,290 +2,162 @@
-    +-------------------------+--------------------------------------------+
-    | ``MNE_TRIGGER_CH_MASK`` | Mask to be applied to the trigger channel  |
-    |                         | values, see :ref:`mne_process_raw`.        |
-    +-------------------------+--------------------------------------------+
-
-
-Troubleshooting
-###############
-
-Missing libXp.so.6
-^^^^^^^^^^^^^^^^^^
-
-If when running an MNE command you get an error about a missing library ``libXp.so.6`` such as:
+An easy way to verify whether your installation of MNE-C is working is to test
+the OpenGL graphics performance:
 
 .. code-block:: console
 
-    $ mne_process_raw
-    mne_process_raw: error while loading shared libraries: libXp.so.6: cannot open shared object file: No such file or directory
+    $ $MNE_ROOT/bin/mne_opengl_test
 
-If you are using Ubuntu Linux and you have admin access to the machine you can do:
+This will render an inflated brain surface repeatedly, rotating it by 5 degrees
+around the z-axis between redraws. The time spent for each full revolution is
+printed to the terminal window where ``mne_opengl_test`` was invoked.  Switch
+focus to that terminal window and use the interrupt key (usually control-c) to
+halt the test.
+
+The best graphics performance occurs when MNE-C renders to a local display on a
+computer with hardware acceleration enabled. The ``mne_analyze`` GUI has a menu
+item "On GLX..." in the Help menu; if the GLX dialog says "Direct rendering
+context" then hardware acceleration is in use. If you are rendering to a local
+display and see "Nondirect rendering context", it is recommended that you
+enable hardware acceleration (consult a search engine or your local IT support
+staff for assistance). If you are rendering to a remote display or using a VNC
+connection, "Nondirect rendering context" is normal.
+
+On the fastest graphics cards, the time per revolution in the
+``mne_opengl_test`` is well below 1 second. If your time per revolution is
+longer than 10 seconds, either the graphics hardware acceleration is not in
+effect or you need a faster graphics adapter.
+
+Troubleshooting MNE-C installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If MNE-C can't find ``libxp.so.6``, download libxp6 from
+`ubuntu <https://packages.ubuntu.com/search?keywords=libxp6>`_ or
+`debian <https://packages.debian.org/search?keywords=libxp6>`_ and install with
+``dpkg`` or similar:
 
 .. code-block:: console
 
-    $ sudo add-apt-repository "deb http://security.ubuntu.com/ubuntu precise-security main"
-    $ sudo apt -o Acquire::AllowInsecureRepositories=true -o Acquire::AllowDowngradeToInsecureRepositories=true update
-    $ sudo apt -o APT::Get::AllowUnauthenticated=true install libxp6
-    $ sudo add-apt-repository -r "deb http://security.ubuntu.com/ubuntu precise-security main"
-    $ sudo apt update
-
-.. _BABDBCJE:
+    $ sudo dpkg -i libxp6_1.0.2-1ubuntu1_amd64.deb
```

Comment:
personally I think the download-and-use-dpkg approach is preferable to asking users to change their `apt` security policies, but I could be wrong.